collectors/logsmanagement.plugin/plugin_logsmanagement_systemd.c

```diff
@@ -0,0 +1,204 @@
+                , RRDSET_TYPE_AREA
+        );
+        for(int j = 0; j < 192; j++){
+            char dim_prior_name[4];
+            sprintf(dim_prior_name, "%d", j);
+            chart_data->dim_prior[j] = rrddim_add(chart_data->st_prior, dim_prior_name, 
+                                                            NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        }
+        chart_data->dim_prior[192] = rrddim_add(chart_data->st_prior, "Unknown", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+    }
+
+    /* Syslog severity level (== Systemd priority) - initialise */
+    if(p_file_info->parser_config->chart_config & CHART_SYSLOG_SEVER){
+        chart_data->st_sever = rrdset_create_localhost(
+                (char *) p_file_info->chart_name
+                , "severity levels"
+                , NULL
+                , "priority"
+                , NULL
+                , "Severity Levels"
+                , "severity levels/s"
+                , "logsmanagement.plugin"
+                , NULL
+                , NETDATA_CHART_PRIO_SYSLOG_SEVER
+                , p_file_info->update_every
+                , RRDSET_TYPE_AREA
+        );
+        chart_data->dim_sever[0] = rrddim_add(chart_data->st_sever, "0:Emergency", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[1] = rrddim_add(chart_data->st_sever, "1:Alert",     NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[2] = rrddim_add(chart_data->st_sever, "2:Critical",  NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[3] = rrddim_add(chart_data->st_sever, "3:Error",     NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[4] = rrddim_add(chart_data->st_sever, "4:Warning",   NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[5] = rrddim_add(chart_data->st_sever, "5:Notice",    NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+        chart_data->dim_sever[6] = rrddim_add(chart_data->st_sever, "6:Informational", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);                                                 
+        chart_data->dim_sever[7] = rrddim_add(chart_data->st_sever, "7:Debug",     NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);     
+        chart_data->dim_sever[8] = rrddim_add(chart_data->st_sever, "Unknown",     NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);  
+    }
+    
+    /* Syslog facility level - initialise */
+    if(p_file_info->parser_config->chart_config & CHART_SYSLOG_FACIL){
+        chart_data->st_facil = rrdset_create_localhost(
+                (char *) p_file_info->chart_name
+                , "facility levels"
+                , NULL
+                , "priority"
+                , NULL
+                , "Facility Levels"
+                , "facility levels/s"
+                , "logsmanagement.plugin"
+                , NULL
+                , NETDATA_CHART_PRIO_SYSLOG_FACIL
+                , p_file_info->update_every
+                , RRDSET_TYPE_AREA
+        );
```

Comment:
Thinking loud, when I took a look at `syslog (3)` I observed that you are not using exactly the same words that we have there. Is it not better to use he same names?

Reply:
I used the names from the standard: https://www.ietf.org/rfc/rfc3164.txt#:~:text=0%20%20%20%20%20%20%20Emergency%3A%20system,debug%2Dlevel%20messages

Reply:
All right, please, add it as reference in documentation, because the first manual that came in my mind was [this](https://man7.org/linux/man-pages/man3/syslog.3.html).